content/release-10-10-0/oee-10-10-0-bundle/10_10_0_3.md

```diff
@@ -0,0 +1,182 @@
+<td style="text-align:left">COE-349</td>
+</tr>
+<tr>
+<td style="text-align:left">UI</td>
+<td style="text-align:left">For configuration of matching rules for calculation profiles, it is now possible to refer back to the length of the calculation interval. This is required for some availability and performance calculations (for example, when calculating the actual production amount based on a flow rate). The <code>Interval Length</code> can now be selected from the drop-down menu in the dialog.</td>
+<td style="text-align:left">COE-541</td>
+</tr>
+<tr>
+<td style="text-align:left">UI</td>
+<td style="text-align:left">The OEE application was updated to Angular 11 and Cumulocity WebSDK 1011.0.15</td>
+<td style="text-align:left">COE-477</td>
+</tr>
+<tr>
+<td style="text-align:left">Microservice</td>
+<td style="text-align:left">Upgraded Apama to 10.7.2.5.</td>
+<td style="text-align:left">COE-556</td>
+</tr>
+</tbody>
+</table>
+
+### Fixes
+
+<table>
+<colgroup>
+    <col style="width: 15%;">
+    <col style="width: 70%;">
+    <col style="width: 15%;">
+</colgroup>
+<thead>
+<tr>
+<th style="text-align:left">Component</th>
+<th style="text-align:left">Description</th>
+<th style="text-align:left">Issue</th>
+</tr>
+</thead>
+<tbody>
+
+<tr>
+<td style="text-align:left">Microservice</td>
+<td style="text-align:left">When using calculation categories, for all categories the formula of the first category was used, leading to incorrect calculation results. The calculation of categories has now been fixed.</td>
+<td style="text-align:left">COE-584</td>
+</tr>
+<tr>
+<td style="text-align:left">Microservice</td>
+<td style="text-align:left">When using line profiles, the subscriptions for MEAs were not created correctly and OEE calculation was not performed. This is issue is now resolved, subscriptions are created for the calculation profiles to fix the OEE calculation of line profiles.</td>
+<td style="text-align:left">COE-532</td>
+</tr>
+<tr>
+<td style="text-align:left">Microservice</td>
+<td style="text-align:left">Formulas defining machine status and quality status conditions were not properly deleted. This led to problems when switching machine status and quality status calculation off. A similar problem occurred when enabling categories on a profile using machine and quality status calculations. This issue has been fixed, the formulas are now deleted correctly.</td>
```

Comment:
switching machine status and quality status calculation off > switching off machine status and quality status calculation

Reply:
Done